aerial_robot_base/launch/external_module/mocap.launch

```diff
@@ -1,13 +1,19 @@
 <launch>
+
+  <arg name="robot_id" default=""/>
```

Comment:
@kanke-hayata 

I think these four lines are redundant, and  following one  line is enough
```
 <arg name="robot_id" default="1"/>
```